nixos/modules/services/x11/desktop-managers/surf-display.nix

```diff
@@ -0,0 +1,47 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.xserver.desktopManager.surf-display;
+
+in {
+  options = {
+    services.xserver.desktopManager.surf-display = {
+      enable = mkEnableOption "Enable surf-display as a kiosk browser session";
```

Comment:
"Enable" is redundant text, already included in `mkEnableOption`.

Reply:
Hmm, good catch, thanks!